clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java

```diff
@@ -244,11 +231,11 @@ private String getStorageRequestPathForSingleKey(byte[] key) {
   }
 
   private String getStorageRequestPath() {
-    return TYPE_STORAGE + "/" + storeName;
+    return TYPE_STORAGE + "/" + getStoreName();
   }
 
   protected String getComputeRequestPath() {
-    return TYPE_COMPUTE + "/" + storeName;
+    return TYPE_COMPUTE + "/" + getStoreName();
   }
```

Comment:
Nitpick: These are supposed to be immutable... should probably pre-allocate them.

Reply:
Fixed